test/utility/FileUtility.h

```diff
@@ -15,9 +15,11 @@
 #ifndef SERIALBOX_UTILITY_FILEUTILITY_H
 #define SERIALBOX_UTILITY_FILEUTILITY_H
 
+#include <boost/core/noncopyable.hpp>
 #include "utility/UnittestEnvironment.h"
-#include <boost/filesystem.hpp>
+#include "serialbox/core/Filesystem.h"

```

Comment:
Yeah!

Reply:
I was using it consistently in my changes! The difference here is that the test is not part of the library that's why relative doesn't make sense.